all_files/backend_not_separated.py

```diff
@@ -63,7 +63,7 @@ def present_schedule(list_of_formatted_entries):
 
 
```

Comment:
Hi @melanieet  , Thanks for the update. I am wondering what is the reason you changed the whole camel case variable names and changed them to _ ?

Reply:
Hi @ellietms,
No worries! I think it's [Python convention](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names) not to use camelcase for variable and function names.

Reply:
@melanieet  no looks good, but I just it's megs codes. I actually  updated the main branch , could you please merge the changes and update the variables (if you would like to change them)